chromadb/api/segment.py

```diff
@@ -174,24 +174,26 @@ def create_collection(
             database=database,
             dimension=None,
         )
+
+        # Get the segment related information for the collection.
+        # NOTE: Below call does not actually create segments in sysdb.
+        # TODO: Move this call out of the manager or change name.
```

Comment:
I think we should just change the name - I agree its confusing. Its just creating the data objects